packages/core/test/forms/asyncControllableInputTests.tsx

```diff
@@ -170,3 +172,149 @@ describe("<AsyncControllableInput>", () => {
         });
     });
 });
+
```

Comment:
You're right that this feels a bit strange since `AsyncControllableTextArea` has deviated from `AsyncControllableInput`. I think it would be fair to use `describe.each` (assuming that part still makes sense), but add an inline source code comment that the plan is to have `AsyncControllableInput` use the hook in a future change? Therefore it makes sense for the test to stay together?

Reply:
Would we still be able to share the tests between the `textarea` and `input` with the `describe.each` approach? I did like the initial version of the PR that did that.